src/main/java/com/deloitte/elrr/services/controller/PersonController.java

```diff
@@ -137,41 +136,44 @@ public class PersonController {
         "Getting %s for Person with id: %s";
 
     /**
+     * Get all persons with optional filters.
      *
-     * @param personId
-     * @param ifi
+     * @param filters Optional filters for person search
      * @return ResponseEntity<List<PersonDto>>
      * @throws ResourceNotFoundException
      */
     @PreAuthorize("hasPermission('person', 'READ')")
     @GetMapping("/person")
     public ResponseEntity<List<PersonDto>> getAllPersons(
-            @RequestParam(value = "id", required = false) final UUID personId,
-            @RequestParam(value = "ifi", required = false) final String ifi) {
-        log.info("getting  PersonDto:.........");
-        log.info("getting Person id:........." + personId);
-        List<PersonDto> persontoList = new ArrayList<>();
-        if (personId != null) {
-            personSvc.get(personId).ifPresent(person -> {
-                PersonDto personDto = mapper.map(person, PersonDto.class);
-                persontoList.add(personDto);
-            });
-        } else if (ifi != null) {
-            Identity ifiIdentity = identitySvc.getByIfi(ifi);
-            if (ifiIdentity != null) {
-                Person person = ifiIdentity.getPerson();
-                persontoList.add(mapper.map(person, PersonDto.class));
-            }
-        } else {
-            Iterable<Person> persons = personSvc.findAll();
-
-            for (Person person : persons) {
-                PersonDto personDto = mapper.map(person, PersonDto.class);
-                persontoList.add(personDto);
-            }
-        }
-
-        return ResponseEntity.ok(persontoList);
+            @ModelAttribute final Person.Filter filters) {
+        log.info("getting PersonDto with filters - id: {}, ifi: {}, "
```

Comment:
kinda beefy for info, set to debug level?

Reply:
it is, though this was just following what a bunch of controllers were doing. I'd be in favor of removing most of the `info` calls in favor of debug or the like